docs/platforms/javascript/common/user-feedback/v7/index.mdx

```diff
@@ -0,0 +1,279 @@
+| `formContentBorderRadius` | `--form-content-border-radius` | `6px`                                     | `6px`                                     | Border radius style for form content (for example: inputs, buttons)                                   |
+
+Here is an example of customizing only the background color for the light theme using the Feedback constructor configuration:
+
+```javascript
+feedbackIntegration({
+  themeLight: {
+    background: "#cccccc",
+  },
+});
+```
+
+Or the same example above, but using the CSS variables method:
+
+```css
+#sentry-feedback {
+  --background: #cccccc;
+}
+```
+
+### Additional UI Customization
+
+Similar to theme customization above, these are additional CSS variables that can be overridden. These aren't supported in the constructor.
+
+| Variable        | Default                                 | Description                                                                       |
+| --------------- | --------------------------------------- | --------------------------------------------------------------------------------- |
+| `--bottom`      | `1rem`                                  | By default the widget has a position of fixed, and is in the bottom right corner. |
+| `--right`       | `1rem`                                  | By default the widget has a position of fixed, and is in the bottom right corner. |
+| `--top`         | `auto`                                  | By default the widget has a position of fixed, and is in the bottom right corner. |
+| `--left`        | `auto`                                  | By default the widget has a position of fixed, and is in the bottom right corner. |
+| `--z-index`     | `100000`                                | The z-index of the widget                                                         |
+| `--font-family` | `"'Helvetica Neue', Arial, sans-serif"` | Default font-family to use                                                        |
+| `--font-size`   | `14px`                                  | Font size                                                                         |
+
+### Event Callbacks
+
+Sometimes it’s important to know when a user started interacting with the feedback form, so you can add custom logging, or start/stop background timers on the page until the user is done.
+
+Pass these callbacks when you initialize the Feedback integration:
+
+```javascript
+feedbackIntegration({
+  onFormOpen: () => {},
+  onFormClose: () => {},
+  onSubmitSuccess: () => {},
+  onSubmitError: () => {},
+});
+```
+
+### Bring Your Own Button
+
+You can skip the default injected button and use your own button to trigger displaying the form. Call `feedback.attachTo()` to have the SDK attach a click listener to your own button. You can additionally supply the same customization options that the constructor accepts (like for text labels and colors).
+
```

Comment:
```suggestion
You can use your own button instead of the default injected button to trigger the form being displayed, by calling `feedback.attachTo()` to have the SDK attach a click listener to your button. You can also supply the same customization options that the constructor accepts (for example, for text labels and colors).

```